openadapt/config.py

```diff
@@ -87,6 +87,7 @@
     "PLOT_PERFORMANCE": True,
     # Calculate and save the difference between 2 neighboring screenshots
     "SAVE_SCREENSHOT_DIFF": False,
+    "SCRUB_PROVIDER_NAME": ["Presidio"],
```

Comment:
How do you feel about having it set up such that you could load the scrubbing class from the names here? For example:
instead of using:
```
scrub = PresidioScrubbingProvider()
```
You could load it like this:
```
scrub = ScrubbingProvider(SCRUB_PROVIDER_NAME[0])
```
which would automatically create an object of the PresidioScrubbingProvider class instead of having the user specify the class themselves.

Reply:
Thank you @AvidEslami.

Our idea was that we have to keep ScrubbingProvider abstract. If we do `scrub = ScrubbingProvider(SCRUB_PROVIDER_NAME[0])`, in that case we will have to initialize the ScrubbingProvider class.
Also we can not instantiate Abstract Classes.

The idea of ScrubbingProvider is to set up a level of abstraction for the Provider classes. I hope this helps to understad=nd why we have different initiations :)

Reply:
Thanks @KrishPatel13 !
Good point, I was thinking maybe something more along these lines:
```
def instantiate_scrubber(scrubber_name):
       if (scubber_name == "Presidio"):
              return PresidioScrubbingProvider()
       elif (scrubber_name == ...):
              return ...Provider()

scrub = instantiate_scrubber(SCRUB_PROVIDER_NAME[0])
```
Let me know if this makes sense or if its unreasonable 👍

Reply:
Ohkay Now I see what you are trying to say :)
Yeah, this is definitely a good option. In my opinion it is excatly the same as PresidioScrubbingProvider() used directly.

My followup question is: then what will be the difference between
directly using PresidioScrubbingProvider() vs using instantiate_scrubber(SCRUB_PROVIDER_NAME[0]) 

given that we know that the name at 0 index is Presidio.  So my next question is what are some pros of instantiate_scrubber (..) over PresidioScrubbingProvider to use that ? Any thoughts :-) ?